gnovm/stdlibs/std/banker.gno

```diff
@@ -126,12 +131,34 @@ func (b banker) IssueCoin(addr Address, denom string, amount int64) {
 	if b.bt != BankerTypeRealmIssue {
 		panic(b.bt.String() + " cannot issue coins")
 	}
+
+	prefix := "/" + CurrentRealm().PkgPath() + ":"
+	if !strings.HasPrefix(denom, prefix) {
+		panic(b.bt.String() + " cannot issue coins with denom not prefixed by current realm pkgPath, use std.CurrentRealm().Denom(denom) to get the correct denom")
+	}
+
+	baseDenom := strings.TrimPrefix(denom, prefix)
+	if !reDenom.MatchString(baseDenom) {
```

Comment:
Can we DRY here by create a function like:

```assertDenom(...)```

to make sure of consistence process

Reply:
9c14a1f :pray: